static/app/views/settings/project/projectFilters/projectFiltersSettings.tsx

```diff
@@ -188,7 +193,7 @@ type Props = {
 
 type State = {
   hooksDisabled: ReturnType<typeof HookStore['get']>;
-} & AsyncComponent['state'];
+};
```

Comment:
I think this still needs to be here no?

Reply:
Ah yes, I was testing something and removed it